halo/attest/voter/voter.go

```diff
@@ -549,6 +549,22 @@ func loadState(path string) (stateJSON, error) {
 		return stateJSON{}, errors.Wrap(err, "unmarshal state path")
 	}
 
+	verify := func(voteSets ...[]*types.Vote) error {
+		for _, votes := range voteSets {
```

Comment:
If it's invalid for any of  `Latest` / `Proposed` etc. to be nil, check that here (but I don't know whether that would be invalid, it seems like it would be fine to me).

Reply:
Not sure what you mean. 

A nil and empty slice is the same, and empty slices are allowed.
Nil votes are invalid, but that is handled in `vote.Verify()` (it doesn't panic)

Reply:
Yeah, my question is basically whether it's valid for any of the voteSets to be nil, and it sounds like that's a yes.